siliconcompiler/core.py

```diff
@@ -3552,6 +3569,19 @@ def pipe(self, flow, plan):
                 self.edge(flow, prevstep, step)
             prevstep = step
 
+    ###########################################################################
+    def __write_task_manifest(self, tool, path=None, backup=True):
+        suffix = self.get('tool', tool, 'format')
+        if suffix:
+            manifest_path = f"sc_manifest.{suffix}"
+            if path:
+                manifest_path = os.path.join(path, manifest_path)
+
+            if backup and os.path.exists(manifest_path):
+                shutil.copyfile(manifest_path, f'{manifest_path}.bak')
```

Comment:
Why add the backup?

Reply:
In the testcase, this file gets overwritten and I wanted to keep a backup of it uncase there were problems with replay script that gets generated. There is something else to inspect.

Reply:
Ah, I originally read this wrong. I thought it was defaulting to always dumping two copies of the same manifest, just realized it's copying out an existing one to avoid overwriting.